test/.tmp/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Can we be more specific here about which files we aim to ignore? Alternatively, this can be solved more neatly by just ignoring `.tmp` on a higher level.

Reply:
This is just a simply workaround to avoid having to create a .tmp dir from the unit test